lib/cog/commands/history.ex

```diff
@@ -65,6 +66,9 @@ defmodule Cog.Commands.History do
     with {:ok, hist_end} <- parse_int(hist_end),
       do: {:ok, {nil, hist_end}}
   end
```

Comment:
Are these function heads that include `"-"` still used? It doesn't look like the parser will parse `-` as a command arg and i couldn't figure out how else this was called.